pages/Hydra.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: Hydra Ecosystem | Hydraecosystem.org
+permalink: /Hydra
+---
+
+# What is Hydra?
+Hydra is a framework that enables REST APIs to be described semantically using RDF. It is based on JSON-LD and proposed as <a href="https://www.hydra-cg.com/spec/latest/core/">W3C draft</a>. It can also be formulated as a documentation framework based on Linked Data which provides vocabulary (similar to schema.org) that aids in the creation of Next Generation Web APIs.
+
+# What's so unique about it?
+
+Hydra in a nutshell is just an extension of JSON-LD technology. As you might already know that our current web is transitioning to Semantic web. In order to acheive that we need to use semantic web tools where ever possible. We can confidently say that Hydra is the most semantically sound tool at the moment when we talk of documentation frameworks. 
+* It is 100% Semantic Web tool as requested by W3C. 
+* It is also directly interoperable with any existing RDF/OWL repository, [(have a look)](https://en.wikipedia.org/wiki/Linked_data#Linking_Open_Data_community_project). 
```

Comment:
Where did you read all this?

Reply:
Apart from "As you might already know that our current web is transitioning to Semantic web. In order to acheive that we need to use semantic web tools where ever possible. We can confidently say that Hydra is the most semantically sound tool at the moment when we talk of documentation frameworks." most information was taken from https://www.hydraecosystem.org/00-Home and some from other sources like https://symfonycasts.com/screencast/api-platform/hydra